lib/helpers/deprecation_helper.rb

```diff
@@ -0,0 +1,12 @@
+module DeprecationHelper
+  def deprecate_method(old_method, new_method)
+    define_method(old_method) do |*args, &block|
+      warn "Warning: `#{old_method}` is deprecated; use `#{new_method}` instead."
```

Comment:
I wanted to DRY up my code and use `deprecation_message(old_name, new_name)` here but I wound up getting `undefined method `deprecation_message'` errors. I _think_ this is because these methods are being extended as class methods in other modules/classes, yet `define_method` was calling `deprecation_message` as if it were an instance method.